File: LeetCode/Easy/0268-missing-number/0268-missing-number.go
func missingNumber(nums []int) int {
    sum := 0
    originalSum := 0
    for i := 0; i < len(nums); i++ {sum += nums[i]}
    for i := 1; i < len(nums)+1; i++ {originalSum += i}
    return originalSum - sum
}

File: LeetCode/Easy/0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = Arrays.stream(nums).sum();
        int originalSum = 0;
        for (int i = 1; i < nums.length + 1; ++i){
            originalSum += i;
        }
        return originalSum - sum;
    }
}